server/webapp/WEB-INF/rails/webpack/views/components/header_panel/index.tsx

```diff
@@ -22,6 +22,7 @@ import * as style from "./index.scss";
 
 export interface Attrs {
   title: m.Children;
+  sectionName: m.Children;
```

Comment:
can we make `sectionName` optional? as all the pages will not have a `sectionName`